### PR TITLE
Fixes #216

### LINKED ISSUE #216: Front End Highlighting: using Disable Styles with combined styles breaks app's styling
If the WordPress css is combined as an optimization (ie. Autoptimize plugin's Aggregate CSS-files option), Disable Styles will break app's styling:

Screenshot 2023-07-28 at 9 58 00 AM

We likely need to inline app's styles. See: https://blog.logrocket.com/the-best-react-inline-style-libraries-comparing-radium-aphrodite-emotion-849ef148c473/

### PR DESCRIPTION
**Pull request by boonedev**
I kept trying to make this hard...

(Autoptimize plugin)
Screenshot 2023-09-21 at 4 31 31 PM

**Before w/styles disabled**
Screenshot 2023-09-21 at 4 32 10 PM

**After w/styles disabled**
Screenshot 2023-09-21 at 4 32 30 PM

### CODE REVIEW

File: src/app/index.js [hunk 1]
```diff
@@ -699,6 +699,25 @@ class AccessibilityCheckerHighlight {
...
 	 */
 	disableStyles() {
+
+		/*
+		If the site compiles css into a combined file, our method for disabling styles will cause out app's css to break.
+		This checks if the app's css is loading into #edac-app-css as expected. 
+		If not, then we assume the css has been combined, so we manually add it to the document.
+		*/
+		if( ! document.querySelector('#edac-app-css') ){
+			debug('css is combined, so adding app.css to page.');
```
Comment (SteveJonesDev) [inline src/app/index.js:709]: Does the build script remove debug calls?

**Review by SteveJonesDev — CHANGES_REQUESTED** [review]
A nice simple solution. Works well, I just have a question.

File: src/app/index.js (hunk 1 above)
Comment (boonedev) [reply src/app/index.js:709]: It doesn't. The  debug and info functions just check for  INFO_ENABLED & DEBUG_ENABLED flag variables before they output anything to console. I think uglify removes console and comments though if you want to go that route?

Comment (SteveJonesDev) [reply src/app/index.js:709]: It's not necessary right now. Maybe something to think about in the future. There might be a performance benefit in doing so.

**Review by SteveJonesDev — APPROVED** [review]

**Pull request merged**